### PR TITLE
add upload command to upload to s3

### PR DESCRIPTION
**Pull request by jordanlewis**

### CODE REVIEW

**Comment by couchand** [comment]
Did you mean to add a file with the definition of `upload`?

**Comment by jordanlewis** [comment]
Indeed I did. It's also on my home laptop... I'll fix that problem when I get home tonight

**Comment by jordanlewis** [comment]
Added in the missing file.

File: main.go [hunk 1]
```diff
@@ -226,6 +226,17 @@ tag 6151ae1.
 	},
 }
 
+var uploadCmd = &cobra.Command{
+	Use:   "upload <testdir> <backend>",
```
Comment (petermattis) [inline main.go]: Might be convenient to make `<backend>` optional and default to `s3`.

**Pull request merged**